reference/volumes.html.markerb

```diff
@@ -44,7 +44,7 @@ Learn more about using volumes:
 
 - How to [add volume storage for your app](/docs/apps/volume-storage/), step-by-step.
 - How to configure the [`mounts` section](/docs/reference/configuration/#the-mounts-section) for volumes in the `fly.toml` Fly Launch configuration file.
-- How to [scale an app with volumes attached]()
+- How to [scale an app with volumes attached](https://fly.io/docs/apps/scale-count/#scale-an-app-with-volumes)
```

Comment:
```suggestion
- How to [scale an app with volumes attached](/docs/apps/scale-count/#scale-an-app-with-volumes)
```
we use relative links in docs

Reply:
I didn't know if I should use the relative link or a specific link that points to where it talks about the words preceding the link, "Scale an app with volumes attached"